test/e2e/pivoting.go

```diff
@@ -340,16 +344,24 @@ func installIronicBMO(ctx context.Context, inputGetter func() installIronicBMOIn
 	}
 }
 
+type IronicDeploymentType string
+
+const (
+	IronicDeploymentTypeLocal     IronicDeploymentType = "local"
+	IronicDeploymentTypeKustomize IronicDeploymentType = "kustomize"
```

Comment:
```suggestion
	IronicDeploymentTypeKustomize IronicDeploymentType = "kustomize"
	IronicDeploymentTypeBMO IronicDeploymentType = "deploy.sh"
```
Lets put both options here later once irso is working we can decide what to remove.

Reply:
I don't understand the difference between Kustomize and deploy.sh. Is it not the same thing? Where do we use what?

Reply:
Sorry my bad I checked the code again, IronicDeploymentType is only used to remove ironic that is deployed by metal3-dev-env, so it's either going to be IrSO or deploy.sh, we don't need Kustomize here. Let's keep `IronicDeploymentTypeBMO IronicDeploymentType = "deploy.sh"` and remove `IronicDeploymentTypeKustomize IronicDeploymentType = "kustomize"`. 
Sorry for the confusion, we do use Kustomize to deploy ironic in some tests but that is not related to this flow.